homeassistant/components/greeneye_monitor/manifest.json

```diff
@@ -11,5 +11,6 @@
   "iot_class": "local_push",
   "loggers": [
     "greeneye"
-  ]
+  ],
+  "config_flow": true
```

Comment:
config_flow should not be set to true since it only imports the entry. once you add user flow, it can be set to true.